python/ray/scripts/scripts.py

```diff
@@ -1862,6 +1866,233 @@ def local_dump(
     )
 
 
+def format_print_logs_index(api_endpoint, node_id, links):
+    def print_section(name, key):
+        if len(links[key]) > 0:
+            print(f"\n{name}")
+            print("----------------------------")
+            [
+                print(
+                    f"{api_endpoint}/api/experimental/logs/file"
+                    f"?node_id={node_id}&log_file_name={log}"
+                )
+                for log in links[key]
+            ]
+
+    for lang in ray_constants.LANGUAGE_WORKER_TYPES:
+        print_section(f"{lang.capitalize()} Core Driver Logs", f"{lang}_driver_logs")
+        print_section(
+            f"{lang.capitalize()} Core Worker Logs", f"{lang}_core_worker_logs"
+        )
+    print_section("Worker Errors", "worker_errors")
+    print_section("Worker Stdout", "worker_outs")
+    print_section("Raylet Logs", "raylet")
+    print_section("GCS Logs", "gcs_server")
+    print_section("Miscellaneous Logs", "misc")
+    print_section("Autoscaler Monitor Logs", "autoscaler")
+    print_section("Runtime Environment Logs", "runtime_env")
+    print_section("Dashboard Logs", "dashboard")
+    print_section("Ray Client Logs", "ray_client")
+
+
+@cli.command()
+@click.argument(
+    "filters",
```

Comment:
I commented on the design doc, but I feel like glob is a better option. So the arg is either glob or match. In this case, we can remove surprising flag like `-f` (I found it surprising when I tried out this API).